### PR TITLE
black staging

### PR DESCRIPTION
**Pull request by mccrindlebrian**
black entire repo

### CODE REVIEW

File: neurons/miners/wiki_agent/agent.py [hunk 1]
```diff
@@ -11,7 +16,12 @@
 from langchain.agents import Tool
 from langchain.agents import initialize_agent
 from langchain.chains import LLMChain
-from langchain.agents import Tool, AgentExecutor, LLMSingleActionAgent, AgentOutputParser
+from langchain.agents import (
+    Tool,
+    AgentExecutor,
+    LLMSingleActionAgent,
+    AgentOutputParser,
+)
```
Comment (p-ferreira) [inline neurons/miners/wiki_agent/agent.py:24]: It seems those imports are duplicated

**Review by p-ferreira — CHANGES_REQUESTED** [review]
Maybe those changes should be merged into pre-staging, not sure if it's the best idea to do a release specific for this in this week, what's your thoughts @mccrindlebrian @steffencruz ?

**Comment by mccrindlebrian** [comment]
@p-ferreira We have already merged a black PR into pre-staging, so this is just additional to reduce headache

File: neurons/miners/wiki_agent/agent.py (hunk 1 above)
Comment (mccrindlebrian) [reply neurons/miners/wiki_agent/agent.py:24]: great catch

**Review by steffencruz — APPROVED** [review]

**Review by p-ferreira — APPROVED** [review]

**Pull request merged**